src/main/java/com/softserve/demo/dto/LocationDTO.java

```diff
@@ -4,11 +4,11 @@
 import lombok.Setter;
 import lombok.ToString;
 
-
 @Getter
 @Setter
 @ToString
```

Comment:
For what you need @ToString here? Do you used it?

Reply:
OK. I will fix it.